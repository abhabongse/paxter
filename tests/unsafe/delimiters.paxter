@python##"
    def foo(text):
        text = flatten(text)
        return f"({text})"
"##\
It is recommended to escape @verb"@" sign by configuring
the symbol command @verb"@@" to render into a single @verb"@" symbol.
Alternatively, to escape @verb"@@", you may write @verb"@@".
To write the above verbatim command, you may write @verb#"@verb"@@""#.
To write the above verbatim command, you may write @verb##"@verb#"@verb"@@""#"##.
To write the above verbatim command, you may write @verb###"@verb##"@verb#"@verb"@@""#"##"###.

Suppose that we define the following function:
@verb###"@python##"
    def foo(text):
        text = flatten(text)
        return f"({text})"
"##"###

Then the code
    @verb####"@foo{@foo#{}}#@foo##{}#}##@foo###{}##}###}"####
will produce
    @foo{@foo#{}}#@foo##{}#}##@foo###{}##}###}
