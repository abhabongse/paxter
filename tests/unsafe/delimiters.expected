To escape @ sign, you may write @"@".
To write the above snippet, you may write @#"@"@""#.
To write the above snippet, you may write @##"@#"@"@""#"##.
To write the above snippet, you may write @###"@##"@#"@"@""#"##"###.
To write the above snippet, you may write @####"@###"@##"@#"@"@""#"##"###"####.

Suppose that we define the following function:
@python##"
    def foo(text):
        text = flatten(text)
        return f"({text})"
"##

Then the code
    @foo{@foo#{}}#@foo##{}#}##@foo###{}##}###}
will produce
    ((})(}#)(}##))
